petl/io/sources.py

```diff
@@ -382,6 +390,35 @@ def register_writer(protocol, handler_class):
 
     _register_handler(protocol, handler_class, _WRITERS)
 
+
+def get_reader(protocol):
+    '''
+    Retrieve the handler responsible for reading from a remote protocol.
+
+    .. versionadded:: 1.6.1
+    '''
+
+    _get_handler(protocol, _READERS)
+
+def get_writer(protocol):
+    '''
+    Retrieve the handler responsible for writing from a remote protocol.
+
+    .. versionadded:: 1.6.1
+    '''
+
+    _get_handler(protocol, _WRITERS)
+
+def get_codec(protocol):
```

Comment:
Is this used anywhere?

Reply:
No. 

`get_codec`:
  -  is not used by `petl` sources as `read_source_from_arg` and `write_source_from_arg` uses `_get_codec_for` with distinct semantics. 
  - is not used by `fsspec` as it uses `compression='infer' ` parameter
  - it would be a natural complement to `get_reader` and `get_writer` helpers, but I'm unsure if it should be left as public without a use case right now.

What's is your opinion here? 
Keep? Remove?

Reply:
Thanks @juarezr. FWIW if you're not using it internally then I'd be inclined to remove it, but happy for you to decide.

Reply:
I'll remove for now.
Thanks